addon/utils/utils.js

```diff
@@ -22,3 +24,43 @@ export function updateSelectedItemsHash (selections, attrs) {
   }
   return _selections
 }
+
+export function normalizeSort (sort) {
+  if (!Ember.isPresent(sort)) {
+    return []
+  }
+  return sort.map(function (item) {
+    let key = item.value
+    let direction = item.direction === ':desc' ? '-' : ''
+    return `${direction}${key}`
+  })
+}
+
+export function listDefaultSort (items, sortProperties) {
```

Comment:
No need to prefix this with list and you should add a jsdoc here or in the README explaining that you're using Ember.compare and link to the documentation http://emberjs.com/api/classes/Ember.Comparable.html#method_compare